src/test/java/com/projetoGerenciamentoCurso/GerenciamentoCurso/Controller/TurmaControllerTest.java

```diff
@@ -31,8 +37,10 @@ private void criarTurmaRetornaSucesso() throws Exception {
 	}
 
 	@Test
-	private void atualizarTurmaRetornaSucesso() throws Exception {
-
+	void atualizarTurmaRetornaSucesso() throws Exception {

```

Comment:
Esse teste falhou:

![image](https://user-images.githubusercontent.com/8150357/107034737-e4b83e80-6795-11eb-8a9e-b5f5c9efa1cc.png)

Erro: **org.springframework.web.util.NestedServletException: Request processing failed; nested exception is org.springframework.orm.jpa.JpaObjectRetrievalFailureException: Unable to find com.projetogerenciamentocurso.gerenciamentocurso.models.Disciplina with id 1; nested exception is javax.persistence.EntityNotFoundException: Unable to find com.projetogerenciamentocurso.gerenciamentocurso.models.Disciplina with id 1**

Reply:
feito

Reply:
O erro ainda ocorre. Não existe uma disciplina com id 1 no banco de dados. Persisti-la no teste.

Reply:
feito

Reply:
Os erros no teste persistem. O aluno com id 1 não está persistido na base de dados... Com isso, ocorre o erro abaixo:

![image](https://user-images.githubusercontent.com/8150357/107226781-4d532580-69f9-11eb-83be-96bc4479ebdd.png)